src/Components/Main/Pages/LoginPage/LoginPageContainer.js

```diff
@@ -18,3 +18,5 @@ export const LoginPageContainer = () => {
     <LoginPage login={loginCallback} isAuth={isAuth} incorrectAuthText={incorrectAuthText} />
   );
 };
+
+export default LoginPageContainer;
```

Comment:
Is there a reason to use default export instead of named export? =)

Reply:
React.lazy works only with default exports.